frontend/src/components/Header/Header.tsx

```diff
@@ -1,15 +1,84 @@
-import React from "react";
+import React, { useEffect, useState, useRef } from "react";
 import { Link } from "react-router-dom";
 
+import Rank from "../Rank/Rank";
+import Social from "../Social/Social"
+
 interface HeaderProps {
     nextExperimentSlug: string | undefined;
     nextExperimentButtonText: string;
     collectionSlug: string;
     aboutButtonText: string;
     showScore: boolean;
+    totalScore: BigInteger;
+    scoreClass: string;
+    scoreLabel: string;
+    noScoreLabel: string;
 }
 
-export const Header: React.FC<HeaderProps> = ({ nextExperimentSlug, nextExperimentButtonText, collectionSlug, aboutButtonText, showScore }) => {
+export const Header: React.FC<HeaderProps> = ({ nextExperimentSlug, nextExperimentButtonText, collectionSlug, aboutButtonText, showScore, totalScore, scoreClass, scoreLabel, noScoreLabel }) => {
+    
+    const social = {
+        'apps': ['facebook', 'twitter'],
+        'message': `I scored ${totalScore} points`,
+        'url': 'wwww.amsterdammusiclab.nl',
+        'hashtags': ["amsterdammusiclab", "citizenscience"]
+    }
+
+    const useAnimatedScore = (targetScore) => {
```

Comment:
Also, aren't we already using a very similar or perhaps even identical hook in the existing `Score` component (`src/components/Score.jsx`)? We might even think about extracting that hook (or a refined version) to a separate file and use the hook in both that and this `Score` component.

Reply:
That's a complete view. The idea here was to just implement the code already in toontje hoger.